### DISCUSSION TITLE
Easier Folder Creation

### DISCUSSION THREAD

**Discussion by karkir0003** [community/community#34825] [Other Feature Feedback, Questions, & Ideas] [answered]
Is it possible for folder creation in the Github UI to be as easy as "clicking the create folder button" (akin to Google Drive)? Having such a feature would increase the accessibility of Github

**Comment by radar-blip** [comment]
No such button exists yet. You just have to create a file first, and add folder before it using `/`
So. Create new file,  foldername `/` filename

Also, it doesn't matter what type of file goes in that folder, as long as it exists.
Reason being: Github doesn't acknowledge empty folders.

**Reply by karkir0003** [reply]
That was the way I had to create a folder in Github, but I just thought it would be easier if the Github UI had a "create a folder button" just like Google Drive to not have to add "/" in filenames and stuff

**Comment by jfullstackdev** [comment, marked as answer]
GitHub is based on Git by design and creating folders similar to what we are very familiar with is not like that in Git, because it's designed mostly for developers, and folder structure is very important, mostly, say, you are using a framework, the folder structure is already there,

but yes, if you are really interested in making it simple, you can create a script for that, but if you are really willing to learn GitHub API, where you can customize that, not necessarily changing the way GitHub functions in folder but through script, you will automate folder creation in a way you want it,

what I'm seeing in the algo is that, in the script, create a folder with a temp file with a pre-filled content, then have it triggered by just one button, so users will think it's just one button click
